ngraph/core/include/ngraph/rt_info.hpp

```diff
@@ -20,6 +20,9 @@ void copy_runtime_info(std::shared_ptr<ngraph::Node> from, ngraph::NodeVector to
 NGRAPH_API
 void copy_runtime_info(const ngraph::NodeVector& from, std::shared_ptr<ngraph::Node> to);
 
+NGRAPH_API
+void copy_runtime_info(const ngraph::NodeVector& from, ngraph::NodeVector to);
+
```

Comment:
why do we need it? looks like the declaration below is the same

Reply:
I think it was left after several mergers with the master.
Removed.